### DISCUSSION TITLE
After updating to the latest release version, unable to sign in with error message "Something went wrong"

### DISCUSSION THREAD

**Discussion by angyen** [microrealestate/microrealestate#166] [Q&A]
Good day.

I need help when after updating from the previous version which need to be "build" to the latest one which doesn't require "build"

I used the same installation instruction as per the homepage and able to reach the login page.

However, when I enter my login credentials, I couldn't login and error message of "Something went wrong" was shown.

I tried restoredb but the same problem still persist.

May I know how do I get my previous data and get the app running?

Thank you in advance for assisting.

**Comment by camelaissani** [comment]
Hello,
What are the logs of the authenticator service? You can run this command to get the logs:
`docker logs $(docker container ls -qf name=authenticator)`

**Reply by vaquicidio** [reply]
hello, sorry to reply back in this post, but im having the same issue  @angyen had.
After running the command i got this:

2024-11-06T20:42:17.1717 <I> Starting Authenticator...
2024-11-06T20:42:19.1919 <I> Authenticator ready and listening on port 8000
2024-11-06T20:44:18.1818 <E> Error: invalid credentials
2024-11-06T20:44:18.1818 <I> POST 403 152ms /landlord/refreshtoken

Any suggestion? thanks

**Comment by camelaissani** [comment]
@vaquicidio I need more information to figure out the issue.
How did you install the application? Is it running on http://localhost:8080/landlord? Have you been able to create a first user?

**Reply by vaquicidio** [reply]
Sure, installed on ubuntu desktop 24.04, im using the new method to run it local with the localhost setup (docker compose --profile local up). And no, i cant create the first user. Thats when the error comes up. Before the update i could run it without problem.

**Comment by camelaissani** [comment]
Could you download the last pushed version of docker-compose.yaml file and restart the application? Are you trying to connect with an account created before the update?

**Reply by vaquicidio** [reply]
I did dowloaded the last version a few hours ago,  restarted and even purged all images and containers, recreate tokens, and tried with new account (not created before update). Thanks for your time.

**Reply by camelaissani** [reply]
Did you try to connect with the user even if there are some errors in the console during the signup?
The first 403 errors are not critical and can be ignored.

**Reply by vaquicidio** [reply]
My bad, ithink i did not explained myself clearly, i didnt even get to create the user,  i can load the landlord signup page but cant create the landlord user account, just getting the "something wrong" error. Think gonna installed an older version of ubuntu and mre because everything used to work. Thanks again for your effort really appreciate.

**Reply by Salmisaid0** [reply]
I have the same problem , i can't create user 
and i don't Know how download the older version of MRE 
Can u help me 
?

**Comment by sbluhm** [comment]
I seem to have something similar on a new install on Fedora 41 (I can't create a new landlord).

Logs on the docker startup that look wrong:

```
root@pc:/tmp/mre# APP_PORT=8080 docker compose --profile local up
WARN[0000] The "SMTP_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "MAILGUN_API_KEY" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SMTP_SECURE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SMTP_USERNAME" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SMTP_PASSWORD" variable is not set. Defaulting to a blank string. 
WARN[0000] The "MAILGUN_DOMAIN" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SMTP_SERVER" variable is not set. Defaulting to a blank string. 
WARN[0000] The "EMAIL_BCC" variable is not set. Defaulting to a blank string. 
WARN[0000] The "APP_DOMAIN" variable is not set. Defaulting to a blank string. 
[+] Running 10/0
 ✔ Container mre-redis-1              Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-mongo-1              Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-tenantapi-1          Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-pdfgenerator-1       Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-emailer-1            Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-authenticator-1      Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-api-1                Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-gateway-1            Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-landlord-frontend-1  Created                                                                                                                                                                                            0.0s 
 ✔ Container mre-tenant-frontend-1    Created                                                                                                                                                                                            0.0s 
Attaching to api-1, authenticator-1, emailer-1, gateway-1, landlord-frontend-1, mongo-1, pdfgenerator-1, redis-1, tenant-frontend-1, tenantapi-1
redis-1              | find: '.': Permission denied
mongo-1              | find: '/data/db': Permission denied
mongo-1              | chown: changing ownership of '/data/db': Permission denied
redis-1              | chown: changing ownership of '.': Permission denied
mongo-1 exited with code 1
redis-1 exited with code 1
tenantapi-1          | 2025-01-12T11:29:22.2222 <I> Starting tenantapi...
emailer-1            | 2025-01-12T11:29:23.2323 <I> Starting Emailer...
authenticator-1      | 2025-01-12T11:29:24.2424 <I> Starting Authenticator...
pdfgenerator-1       | 2025-01-12T11:29:24.2424 <I> Starting PdfGenerator...
pdfgenerator-1       | (node:1) NOTE: The AWS SDK for JavaScript (v2) will enter maintenance mode
pdfgenerator-1       | on September 8, 2024 and reach end-of-support on September 8, 2025.
pdfgenerator-1       | 
pdfgenerator-1       | Please migrate your code to use AWS SDK for JavaScript (v3).
pdfgenerator-1       | For more information, check blog post at https://a.co/cUPnyil
pdfgenerator-1       | (Use `node --trace-warnings ...` to show where the warning was created)

```

Root case was SELINUX in my case. I set it to permissive for now.

**Comment by Namtarux** [comment]
Hi 
is this issue resolved now as i just ran into the same issue and following the comments here did not seem to get a way to resolve it.
thanks
